common/src/IO/ObjSerializer.h

```diff
@@ -117,7 +118,7 @@ namespace TrenchBroom {
             Object m_currentObject;
             ObjectList m_objects;
         public:
-            explicit ObjFileSerializer(const Path& path);
+            explicit ObjFileSerializer(const Path& path, const Path& gamepath);
```

Comment:
If you add a second parameter to the constructor, you can drop the `explicit` keyword.